xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-validation/xwiki-platform-model-validation-default/src/main/resources/ApplicationResources_de.properties

```diff
@@ -1,24 +0,0 @@
-# ---------------------------------------------------------------------------
-# See the NOTICE file distributed with this work for additional
-# information regarding copyright ownership.
-#
-# This is free software; you can redistribute it and/or modify it
-# under the terms of the GNU Lesser General Public License as
-# published by the Free Software Foundation; either version 2.1 of
-# the License, or (at your option) any later version.
-#
-# This software is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-# Lesser General Public License for more details.
-#
-# You should have received a copy of the GNU Lesser General Public
-# License along with this software; if not, write to the Free
-# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
-# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
-# ---------------------------------------------------------------------------
-entitynamevalidation.ReplaceCharacterEntityNameValidation.name=Zeichenersetzung
-entitynamevalidation.ReplaceCharacterEntityNameValidation.usage=Einige Zeichen sind bei dieser Strategie verboten und sollten ersetzt oder entfernt werden. Erkundige Dich bei deinem Administrator nach der genauen Liste der verbotenen Zeichen und nach dem zu verwendenden Ersatz.
-## Missing: entitynamevalidation.ReplaceCharacterEntityNameValidation.usage=Some characters are forbidden with this strategy and should be replaced or removed. Ask your administrator to know exactly the list of forbidden characters, and the replacement to use.
-entitynamevalidation.SlugEntityNameValidation.name=Vorformatierte Namen (keine Akzente, kein Leerzeichen, keine Sonderzeichen)
-entitynamevalidation.SlugEntityNameValidation.usage=Alle Sonderzeichen, einschlie\u00DFlich Leerzeichen und Akzente, sind verboten.
```

Comment:
It seems surprising to delete this translation file in this PR.

Reply:
Does not make sense to me too. Maybe something went bad during a rebase.